src/components/modal_content.jsx

```diff
@@ -1,9 +0,0 @@
-import React from "react";
```

Comment:
removed as this component was adding nothing, it was rendering what it got it props